upgrading.md

```diff
@@ -0,0 +1,170 @@
+# Upgrading Philanthropy Data Commons dependencies
+
+This document guides one through upgrades for the back-end services at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has its own process.
+
+This applies to the non-PDC-web-container services. As of this writing that is
+the auth, database, and reverse-proxy services. The rest is either in the PDC
+web service, such as node dependencies managed in the `service` project, or on
+the host, such as cert-bot.
+
+See the summary in `deploying.md` regarding how software gets deployed. The same
+explanation regarding compose.yml[1] applies.
+
+In general, try it on a local development environment first, then make sure it
+succeeds in the test environment, then finally do it in production.
+
+[1]: https://github.com/PhilanthropyDataCommons/deploy/blob/main/compose.yml
+
+## Upgrade postgresql (minor version)
+
+Caution: do not use "latest". That won't work the way we want. Do not use a
+non-specific version such as 14-debian-11 or 14.8.0. Use only specific versions.
```

Comment:
How is "14.8.0" not a specific version?

Reply:
There may be multiple images containing a 14.8.0 version of postgresql built and pushed to docker hub. So while it appears specific, and is a specific version of postgresql, we are working with particular built images and can specify exactly that image. I'll update the text to indicate "less specific" versus "more specific."